benchmarks/bench_explicitOperators.cpp

```diff
@@ -114,7 +114,10 @@ TEST_CASE("DivOperator")
                 NeoN::fill(nfDivT.boundaryData().value(), 0.0);
                 fvcc::GaussGreenDiv<NeoN::scalar>(exec, nfMesh, scheme)
                     .div(nfDivT, nfPhi, nfT, dsl::Coeff(1.0));
-                Kokkos::fence();
+                if (execName == "GPUExecutor")
+                {
+                    Kokkos::fence();
```

Comment:
I guess we could move that to the NeoN::Executor so that we can do `exec.fence()` which would be a noop on CPUs.